teams/vsa/onboarding/VSA Technical Orientation.md

```diff
@@ -1,354 +1,489 @@
 # VSA Technical Orientation
+
 Technical orientation from the VSA Technical Lead ([pshort@governmentcio.com](mailto:pshort@governmentcio.com)/[paul.short@va.gov](mailto:paul.short@va.gov)) as of 05/05/2020. Primary audience are VSA front end and back end developers, but other VSA team members may find some of the links here useful. There is a lot of decent documentation available; however, some of it is changing, redundant, outdated, or requires more context in order to navigate. This is an attempt to capture the most important links in a structured order for a new developer.
 
+## Table of Contents
```

Comment:
A TOC is a great idea. Are you using a VS Code Plugin to help generate these from ##, or is this manual for now?